### ISSUE TITLE
Client Friendly Postman Collection & Instruction

### ISSUE THREAD

**Issue by mjones-oddball** [department-of-veterans-affairs/notification-api#1539]
## User Story - Business Need

Prospective clients often want to try out VA Notify to get a feel for it before building their notification triggers. We share our full postman collection with them today, which includes many more endpoints than are relevant to them. This causes confusion and results in engineering spending time troubleshooting. The goal here is to offer a revised postman collection and instruction so that our partners can easily try VA Notify on their own time.

**Product will be QAing this ticket**
- [x] Ticket is understood, and QA has been contacted (if the ticket has a QA label).

### User Story(ies)

**As a** a VA business
**I want** to try out VA Notify
**So that** I can determine if this is a right fit for my use case

### Additional Info and Resources

- Areas of confusion:
    - What endpoints to use
    - What variables to populate
    - Which collection to use
- Current tips for postman use are at the bottom of our [playbook](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/va-notify#playbook).

## Engineering Checklist

- [x] Create a new collection folder for clients in notification-api
- [x] Create collection(s) paired down to the variables and routes in `v2/notifications`
    - [x] Please give the collection a new name so it doesn't overwrite our full collection
    - [x] Pre-populate any variables that can be, e.g. urls
- [x] Draft a readme in this folder with information on how to get started using postman.
- [x] Alert Product to "QA" when ready

## Acceptance Criteria

- [x] Client can use postman to test against the Staging or Production API routes, however, only Staging env will be supplied to clients in an effort to protect Prod from accidental/misuse.
    - [x] Environment variables list is limited to what's required to execute the routes below
- [x] Client can execute all routes under `v2/notifications` in postman
    - [x] Send an email with email address
    - [x] Send an email with recipient identifier
    - [x] Send an SMS with email address
    - [x] Send an SMS with recipient identifier
    - [x] Send a push notification
    - [x] Get a notification status by id

## QA Considerations
_For QA to populate. Leave blank if QA is not applicable on this ticket._

**Product will be QAing this ticket**
Can follow the readme to get started with the new collection(s)

## Out of Scope
_Leave blank if n/a_

This isn't intended to overhaul anything within the current `v2/notifications` routes. We just want to take what we have and pair it down, so clients aren't confused by endpoints and variables that they shouldn't be using.

**Comment by mjones-oddball** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/VA-Notify-620d21369d810a00146ed9c8/issues/department-of-veterans-affairs/notification-api/1539?planning-poker) with Zenhub @cris-oddball @edDocMe360 @EvanParish @k-macmillan @kalbfled @ldraney @nikolai-efimov

**Comment by ldraney** [comment]
Kyle suggests copying a collection in Postman, and then deleting the unnecessaries for the customer.

**Comment by mjones-oddball** [comment]
I asked Ed not to worry about testing the push notification endpoint because the set up is too steep for today. I can do that on my end when it's ready.

**Comment by mjones-oddball** [comment]
@edDocMe360 Please update the ticket with your progress

**Comment by edDocMe360** [comment]
Put together the collection. Got IDs from Cris. Testing, testing. 

Today, I'm going to put together the instructions.

**Comment by mjones-oddball** [comment]
@edDocMe360 where are you thinking of storing the collection in the api repo and what are you going to name it? Want to make sure @k-macmillan reviews and OKs it

**Comment by edDocMe360** [comment]
@mjones-oddball, I was going to ask that very question at stand-up.

**Comment by mjones-oddball** [comment]
@edDocMe360 can you post an update here please? Also don't forget to check off the "Ticket is understood, and QA has been contacted" checkbox since we synced at the start of the ticket.

**Comment by edDocMe360** [comment]
@mjones-oddball Moved the file around and got a start on the README.

**Comment by mjones-oddball** [comment]
@npmartin-oddball added the off-track label because we've exceeded 2 pts

**Comment by mjones-oddball** [comment]
Open question about if the PR will need 2 dev reviews or not given the nature of the work. @edDocMe360 

@edDocMe360 don't forget to update the description when you're done and move the ticket to the applicable pipeline. If it needs PR reviews then it will stay in Dev Testing until that is complete. If not, then it should go to the QA lane when the PR and ticket is updated and ready.

My update this AM: I started taking a cursory look at the files/readme and sent Ed some change requests RE: env should be Staging not Dev, variables that should be removed, and readme feedback. I'm awaiting changes before continuing.

**Comment by cris-oddball** [comment]
@mjones-oddball just noting that the ticket description asks for both Staging and Production.

**Comment by mjones-oddball** [comment]
@cris-oddball AC updated: `Client can use postman to test against the Staging or Production API routes, however, only Staging env will be supplied to clients in an effort to protect Prod from accidental/misuse.`

**Comment by mjones-oddball** [comment]
On standup it was decided that this should go through PR review per usual. @edDocMe360 please post in the engineering channel when ready for reviews and let me know when it's approved and ready for me to QA :) CC @npmartin-oddball @k-macmillan

**Comment by cris-oddball** [comment]
PR approved and merged, tag cut and sent to Perf. Would close this ticket but waiting on the checklist to be completed.

@edDocMe360 I cannot close this ticket until you've completed the checklist in this ticket.

**Comment by mjones-oddball** [comment]
Ed updated the description and we confirmed AC was good to go
